public/views/viewBuildError.html

```diff
@@ -9,6 +9,18 @@
     <link rel="stylesheet" type="text/css" href="css/jquery-ui-1.10.4.css" />
     <link rel="stylesheet" type="text/css" href="css/jqModal.css" />
     <link rel="stylesheet" type="text/css" ng-href="{{cssfile}}" />
+    <link rel="stylesheet" href="css/bootstrap.min.css"/>
+    <style type="text/css">
```

Comment:
I debated it, this was an oddly specific use case. This isn't so much adding styles as it is negating bootstrap styles (had to bring in bootstrap for pagination styles, it brought its own rules in for code/tables).